stdlib/public/Concurrency/Clock.cpp

```diff
@@ -132,8 +124,10 @@ switch (clock_id) {
       *seconds = continuous.tv_sec;
       *nanoseconds = continuous.tv_nsec;
 #elif defined(_WIN32)
+      LARGE_INTEGER freq;
+      QueryPerformanceFrequency(&freq);
       *seconds = 0;
-      *nanoseconds = 1000;
+      *nanoseconds = 1000000000 / freq.QuadPart;
```

Comment:
`QueryPerformanceFrequency()` never reports zero post-Windows XP. If we're concerned about that, we can check.